### ISSUE TITLE
Can't blacklist Soulus bone drops

### ISSUE THREAD

**Issue by Vyraal1** [Yuudaari/soulus#43]
I have Silent's Gem in the pack, and it doesn't seem to recognize most of the bones as possible uses for tools, so I've decided to remove most of the bone variants Soulus adds with by editing the [yes_creatures_yes_drops.json](https://hastebin.com/ibamixafec.json). 

Is this the correct / intended way to remove the bone drops? I know t his invalidates part of the essence/bone parts of the mod, but given the loot tables and use of normal bones in other parts of the pack, this was the best way I could manage.

**Comment by Yuudaari** [comment]
I'm not sure I understand the point of this. The only bones that don't have the `bone` ore dictionary is nether bones and ender bones, all other bones have the `bone` ore dictionary. It's intentional that they don't work in recipes that require "bones" because they're not functionally the same as normal bones. The bones with that ore dictionary also can produce normal bonemeal. Their only difference is the chunks they give. If Silent's Gems doesn't work with the `bone` ore dictionary, I recommend opening an issue over there.

Otherwise, yes, this will do what you're trying to do.

**Comment by Vyraal1** [comment]
Hi so after some testing I can't seem to get the blacklisting drops to work correctly - given this file directory:
![image](https://user-images.githubusercontent.com/26587049/37247490-ad286186-2489-11e8-9704-ce451cac8b89.png)

I had profiles switched a couple of times, and used `/soulus reload` ingame to see if I could adjust the drops. it seems that the blacklist/whitelist drop options failed to work each time. I tested it with both Skeletons / Wither Skeletons with the provided JSONS. I'm pretty sure they'are all valid (used Sublime  and online JSON to verify if they're valid). 

[interactionCustom.json](https://hastebin.com/azozurexij.json) (made for testing purpose)
[no_creatures_no_drops.json](https://hastebin.com/kosiqigezi.json) (un-edited)
[yes_creatures_yes_drops](https://hastebin.com/ifoyegovoj.json) (same json as above, with edits)

[yes_creatures_no_drops](https://hastebin.com/baratuyaxa.json) 
So this seemed to be the only JSON that had an affect on drops. The way I tested was by creatively giving myself skeleton spawn eggs from JEI menu and using those to spawn skeletons. I also tested with a Draconic Evolution mob spawner - the skeletons made from this dropped arrows, minecraft bones, etc. 

tldr; I'm confused again sorry.

**Comment by Yuudaari** [comment]
Can you post your creatures.json as well? I want to repro your environment

**Comment by Yuudaari** [comment]
Oh actually yea this is a bug, I just realised why it happens, I’ll add it to my todo. Not sure how long it’s going to take me to get to it tho, as my job has gotten pretty busy recently. Hopefully within the week

**Comment by Vyraal1** [comment]
Okay as long as I'm not crazy, I thought I was going crazy.

**Comment by Yuudaari** [comment]
[v3.4.1](https://github.com/Yuudaari/soulus/releases/tag/v3.4.1)
